templates/aws-stack.yml

```diff
@@ -626,10 +632,13 @@ Rules:
             - !Equals
               - !Ref PipelineSigningKMSKeyId
               - ""
+            - !Equals
+              - !Ref PipelineSigningKMSKeyARN
```

Comment:
```suggestion
              - !Ref PipelineSigningKMSKeyArn
```

I guess we should probably follow the convention in CloudFormation to only capitalise the first letter of an acronym?

Reply:
Correct - have updated this locally

Reply:
I had it lowercase but changes to "ARN" to follow the pattern set by Buildkite in this stack. I also would prefer "Arn" but this would be a snowflake in this stack

Reply:
Ah you're right, sorry!